src/scripts/Dockerfile.android

```diff
@@ -0,0 +1,22 @@
+FROM debian:stretch AS debian
+ARG SDK
+RUN apt-get update && apt-get -y --no-install-recommends install \
+	libboost-all-dev \
```

Comment:
Is boost being used here? I don't think we really need it in this context since mostly it just enables asio support in the CLI utils.

Reply:
That s a good point, the lesser dependencies the better.